tests/utils/test_transcript_annotation.py

```diff
@@ -0,0 +1,157 @@
+"""Tests for the transcript_annotation module."""
+
+import hail as hl
+import pytest
+
+from gnomad.utils.transcript_annotation import (
+    clean_tissue_name_for_browser,
+    create_tx_annotation_by_region,
+)
+
+
+class TestCleanTissueNameForBrowser:
+    """Tests for the clean_tissue_name_for_browser function."""
+
+    @pytest.mark.parametrize(
+        "input_name,expected_output",
+        [
+            ("BasalGanglia", "basal_ganglia"),
+            ("NucleusAccumbens", "nucleus_accumbens"),
+            ("SpinalCord", "spinal_cord"),
+            ("CervicalC", "cervical_c"),
+            ("SubstantiaNigra", "substantia_nigra"),
+            ("CulturedFibroblasts", "cultured_fibroblasts"),
+            ("LowerLeg", "lower_leg"),
+            ("TransformedLymphocytes", "transformed_lymphocytes"),
+            ("AnteriorCingulateCortex", "anterior_cingulate_cortex"),
+            ("BA24", "ba24"),
+            ("BA9", "ba9"),
+            ("EBV", "ebv"),
```

Comment:
shouldn't one of these tests be checking for an underscore in the name (like `b_a24`)?